Content/posts/how-we-do-vapor-releases.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2025-08-14 18:00
+description: See how we approach releases in Vapor, so we can release quickly and often to save time and provide a better experience for users and contributors.
+tags: Framework, Automation
+author: Tim
+authorImageURL: /author-images/tim.jpg
+---
+# How we approach releases at Vapor
+
+I've had some interesting conversations recently at why we do releases the way we do in Vapor so thought it would be worth sharing. 
+
+## Vapor Releases
+
+Our release process is generally pretty simple. For everything outside of new major versions we treat each pull request as a new release. Each pull request will get tagged with one of three labels:
+
+* `no-release-needed` - this usually encompasses things like test changes, updates to CI or the README or updates to governance files. Basically anything that doesn't touch `Sources` and wouldn't result in any changes for people pulling in the code. 
+* `semver-patch` - in following with SemVer guidelines this would be bug fixes that don't have an effect on the public API. 
+* `semver-minor` - changes that would introduce new APIs that can be adopted. 
```

Comment:
Maybe add a bit about major releases too?

Reply:
Major releases were explicitly excluded by the previous paragraph; the `semver-major` label basically never sees actual use.